src/connection-manager/connections.ts

```diff
@@ -55,11 +62,35 @@ async function handleWsMessageTypes(
     if (ledger_hashes.includes(validationData.ledger_hash)) {
       validationData.networks = networks
     }
-    void agreement.handleValidation(data as ValidationRaw)
+
+    // Get network of the validation if ledger_hash is not in cache.
```

Comment:
This comment is confusing - there's no ledger_hash check

Reply:
NVM I see the line of code above.

Reply:
Seems like it's more the opposite - use the ledger hash data if it's not in the validators db.